src/core/midi/MidiPort.cpp

```diff
@@ -238,6 +267,7 @@ void MidiPort::loadSettings( const QDomElement& thisElement )
 	m_outputControllerModel.loadSettings( thisElement, "outputcontroller" );
 	m_fixedInputVelocityModel.loadSettings( thisElement, "fixedinputvelocity" );
 	m_fixedOutputVelocityModel.loadSettings( thisElement, "fixedoutputvelocity" );
+	m_fixedOutputNoteModel.loadSettings( thisElement, "fixedoutputnote" );
```

Comment:
The coding conventions comment made about also applies here. (NB. The surrounding code may not always follow the coding conventions, but it is generally wise to match it when you are adding new code, or modifying already existing code.)